### PR TITLE
New websites

### PR DESCRIPTION
**Pull request by 3xp0rt**

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -1017,6 +1028,16 @@
...
         "cat" : "news"
        },
+       {
+        "name" : "Community Adobe",
+        "uri_check" : "https://community.adobe.com/t5/forums/searchpage/tab/user?q={account}",
+        "e_code" : 200,
+        "e_string" : "UserSearchItemContainer",
+        "m_string" : "No search results found.",
+        "m_code" : 200,
+        "known" : ["test"],
```
Comment (WebBreacher) [inline wmn-data.json:1038]: Please add a second known user like `janet`.

File: wmn-data.json [hunk 2]
```diff
@@ -1017,6 +1028,16 @@
...
+        "cat" : "techc"
```
Comment (WebBreacher) [inline wmn-data.json:1039]: This should be `tech`

File: wmn-data.json [hunk 3]
```diff
@@ -1839,6 +1860,17 @@
         "known" : ["evdokia", "irina"],
         "cat" : "social"
        },
+       {
+        "name" : "Folkd",
+        "uri_check" : "https://www.folkd.com/?app=core&module=system&controller=ajax&do=usernameExists&input={account}",
```
Comment (WebBreacher) [inline wmn-data.json:1865]: Please consider using a different `uri_check` like `https://www.folkd.com/search/?q={account}&quick=1&type=core_members` as it gives the link to the profile of the person in the results.

File: wmn-data.json [hunk 4]
```diff
@@ -2069,11 +2111,12 @@
         "name" : "Giphy",
         "uri_check" : "https://giphy.com/channel/{account}",
```
Comment (WebBreacher) [inline wmn-data.json:2112]: I got a redirect from this `uri_check` to `https://giphy.com/{account}` (no channel)

File: wmn-data.json [hunk 5]
```diff
@@ -2069,11 +2111,12 @@
         "name" : "Giphy",
         "uri_check" : "https://giphy.com/channel/{account}",
         "e_code" : 200,
-        "e_string" : "Share on GIPHY</title>",
-        "m_string" : "<title>404 Not Found</title>",
+        "e_string" : "\\\"user_id\\\"",
+        "m_string" : "404 Not Found",
         "m_code" : 404,
```
Comment (WebBreacher) [inline wmn-data.json:2116]: `m_code` now is `301`. Please confirm.

File: wmn-data.json [hunk 6]
```diff
@@ -2728,6 +2771,17 @@
...
         "cat" : "social"
        },
+       {
+        "name" : "Instagram (Imginn)",
+        "uri_check" : "https://imginn.com/{account}/",
+        "e_code" : 200,
+        "e_string" : "userinfo",
+        "m_string" : "page-error notfound",
+        "m_code" : 404,
+        "known" : ["instagram", "ramarim"],
```
Comment (WebBreacher) [inline wmn-data.json:2781]: Consider changing `instagram` for another user like `therock`

File: wmn-data.json [hunk 7]
```diff
@@ -3507,6 +3561,16 @@
         "known" : ["daniel", "franco"],
         "cat" : "gaming"
        },
+       {
+        "name" : "MODX.im",
+        "uri_check" : "https://modx.evo.im/profile/{account}/",
+        "e_code" : 200,
+        "e_string" : "class=\"profile\"",
+        "m_string" : "content-error",
```
Comment (WebBreacher) [inline wmn-data.json:3569]: Maybe change this to `class=\"content-error\"` to match above `e_string`

File: wmn-data.json [hunk 8]
```diff
@@ -5604,6 +5689,22 @@
...
+        "uri_check" : "https://www.tindie.com/accounts/check_username/",
+        "uri_pretty": "https://www.tindie.com/stores/{account}/",
+        "post_body": "username={account}",
+        "e_code" : 200,
+        "e_string" : "\"valid\": false",
+        "m_string" : "\"valid\": true",
+        "m_code" : 200,
+        "known" : ["tehrabbitt", "dekunukem"],
+        "cat" : "social",
+        "protection": ["shopping"],
```
Comment (WebBreacher) [inline wmn-data.json:5703]: We do not have a `shopping` protection.

**Review by WebBreacher — CHANGES_REQUESTED** [review]
please review changes and resubmit.

File: wmn-data.json (hunk 3 above)
Comment (3xp0rt) [reply wmn-data.json:1865]: To reduce network traffic, it would be better to add this to the `uri_pretty` parameter.

File: wmn-data.json (hunk 4 above)
Comment (3xp0rt) [reply wmn-data.json:2113]: As I understand it, there are categories for both "channel" and "no channel" (though I'm not sure of the exact terminology). I've updated the `known` parameter with usernames that are channels. We'll need to create a "No channel" category in the future as well.

**Review by WebBreacher — APPROVED** [review]

**Pull request merged**